test/src/test/java/org/corfudb/integration/WorkflowIT.java

```diff
@@ -50,42 +46,42 @@ public void AddNodeIT() throws Exception {
                 .setSingle(true)
                 .runServer();
 
-        CorfuRuntime n1Rt = new CorfuRuntime(getConnectionString(n1Port)).connect();
+        // start a second node
+        final int n2Port = 9001;
+        new CorfuServerRunner()
+                .setHost(host)
+                .setPort(n2Port)
+                .setSingle(false)
+                .runServer();
+
+        // start a third node
+        final int n3Port = 9002;
+        new CorfuServerRunner()
+                .setHost(host)
+                .setPort(n3Port)
+                .setSingle(false)
+                .runServer();
+
+        CorfuRuntime n1Rt = new CorfuRuntime(getConnectionString(n1Port)).setCacheDisabled(true).connect();
 
         CorfuTable table = n1Rt.getObjectsView()
                 .build()
                 .setType(CorfuTable.class)
                 .setStreamName(streamName)
                 .open();
 
-        final int numEntries = 12_000;
+        final int numEntries = 30_000;
```

Comment:
Please use a parameter instead of choosing an arbitrary number

Reply:
Ok.